CIPs/cip-0041.md

```diff
@@ -0,0 +1,79 @@
+---
+cip: 0041
+title: E Hardfork
+author: Joshua Gutow <@trianglesphere>
+discussions-to: TBD
+status: Proposed
+type: Meta
+created: 2021-08-11
+license: Apache 2.0
+---
+
+This is the Meta CIP for the technical specifications of the Celo E Hardfork.
+
+## Simple Summary
+
+Enable the activation of the Celo E Hardfork specification which includes Ethereum Berlin EIPs, Ethereum London EIPS, and Celo CIPs.
+
+## Abstract
+
+Add support for protocol-impacting changes introduced in the Ethereum Berlin and London hardforks and Celo improvement proposals in a hardfork
+named by the community as _"E"_.
+
+This hardfork contains all applicable EIPs in Berlin and London. Some EIPs have been modified to work with the Celo network.
+
+This document proposes the following blocks at which to implement these changes in the Celo networks:
+- `TBD` on Baklava Testnet
+- `TBD` on Alfajores Testnet
+- `TBD` on Celo Mainnet
+
+For more information on the opcodes and their respective EIPs and CIP implementations, please see the _Specification_
+section of this document.
+
+## Motivation
+
+The enhance the EVM's capabilities and while pushing forward the Celo blockchain functionality in order to enable economic conditions for prosperity as well as adopting Ethereum's Berlin and London specifications.
+
+## Specification
+
+The "E" Hardfork specification meta document includes the following proposals:
+
+From Berlin
+* [EIP-2565: ModExp Gas Cost](https://eips.ethereum.org/EIPS/eip-2565)
+* [EIP-2929: Gas cost increases for state access opcodes](https://eips.ethereum.org/EIPS/eip-2929)
+* [EIP-2718: Typed Transaction Envelope](https://eips.ethereum.org/EIPS/eip-2718)
+* [EIP-2930: Optional access lists](https://eips.ethereum.org/EIPS/eip-2930)
+
+From London
+* [EIP-1559: Fee market change for ETH 1.0 chain](https://eips.ethereum.org/EIPS/eip-1559)
+* [EIP-3198: BASEFEE opcode](https://eips.ethereum.org/EIPS/eip-3198)
+* [EIP-3529: Reduction in refunds](https://eips.ethereum.org/EIPS/eip-3529)
+* [EIP-3541: Reject new contracts starting with the 0xEF byte](https://eips.ethereum.org/EIPS/eip-3541)
+
+Modifications to the EIPs
+* EIP-3554: Difficulty Bomb Delay (Included in London) will not be included in the "E" Hardfork
```

Comment:
Please add links to all EIPs that won't be included in this hardfork for easier reference